tests/interfaces/pygame_adapter/rendering/test_grid_renderer.py

```
@@ -0,0 +1,105 @@
+"""Tests for the grid renderer."""
+from unittest.mock import Mock, patch
+
+import pygame
+import pytest
+
+from src.domain.entities.grid import HexGrid
+from src.domain.value_objects.grid_dimensions import GridDimensions
+from src.domain.value_objects.grid_position import GridPosition
```

Ruff (0.8.2)

`src.domain.value_objects.grid_position.GridPosition` imported but unused

Remove unused import: `src.domain.value_objects.grid_position.GridPosition`

(F401)